pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -136,6 +139,52 @@ func (m *managerImpl) IsUnderDiskPressure() bool {
 	return hasNodeCondition(m.nodeConditions, api.NodeDiskPressure)
 }
 
+func (m *managerImpl) createThresholdNotification(diskInfoProvider DiskInfoProvider, podFunc ActivePodsFunc, observations signalObservations) error {
+	// default to no-op notifier
+	m.thresholdNotifier = NewNoOpThresholdNotifier()
+
+	thresholds := m.config.Thresholds
+	if len(thresholds) == 0 {
+		return nil
+	}
+
+	for _, threshold := range thresholds {
+		// only enable memcg threshold notification if a hard memory eviction limit is set
+		if threshold.Signal != SignalMemoryAvailable || threshold.GracePeriod != time.Duration(0) {
+			continue
+		}
+
+		observed, found := observations[SignalMemoryAvailable]
+		if !found {
+			continue
+		}
+
+		quantity := getThresholdQuantity(threshold.Value, observed.capacity, resource.DecimalSI).String()
```

Comment:
the issue is that, when writing to the ```event_control```, it understand 5M but not 5Mi.  am i doing it wrong?  i am doing a ```.String()``` on it.